hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/s3/multipart/TestS3MultipartUploadCompleteResponseWithFSO.java

```diff
@@ -276,4 +278,9 @@ private void addS3MultipartUploadCommitPartResponseFSO(String volumeName,
 
     omMetadataManager.getStore().commitBatchOperation(batchOperation);
   }
+
+  @Override
```

Comment:
@aryangupta1998 

Please overridde getBucketLayout to FSO in S3MultipartUploadCompleteResponseWithFSO.java. This will make TestS3MultipartUploadCompleteResponseWithFSO test happy.

 ```
 @Override
  public BucketLayout getBucketLayout() {
    return BucketLayout.FILE_SYSTEM_OPTIMIZED;
  }
```

Reply:
Done